lib/landrush/server.rb

```diff
@@ -67,16 +67,38 @@ def self.upstream
     def self.start
       ensure_ruby_on_path
       if Vagrant::Util::Platform.windows?
-        # Need to handle Windows differently. Kernel.spawn fails to work, if the shell creating the process is closed.
+        # Need to handle Windows differently. Kernel.spawn fails to work, if
+        # the shell creating the process is closed.
         # See https://github.com/vagrant-landrush/landrush/issues/199
+        #
+        # Note to the Future: Windows does not have a
+        # file handle inheritance issue like Linux and Mac (see:
+        # https://github.com/vagrant-landrush/landrush/issues/249)
+        #
+        # On windows, if no filehandle is passed then no files get
+        # inherited by default, but if any filehandle is passed to
+        # a spawned process (as we are doing) then all files that are
```

Comment:
`as we are doing` - what file handles are we passing to the spawned process on windows?